uvls/src/smt/smt_lib.rs

```diff
@@ -155,151 +155,186 @@ impl SMTModule {
+                            stack.push(CExpr::End);
+                            for i in v.iter().rev() {
+                                stack.push(CExpr::Expr(i));
                             }
                         }
-                        //Args
-                        match e {
-                            Expr::Add(v)
-                            | Expr::Sub(v)
-                            | Expr::Mul(v)
-                            | Expr::Div(v)
-                            | Expr::Or(v)
-                            | Expr::And(v)
-                            | Expr::Implies(v)
-                            | Expr::AtLeast(_, v)
-                            | Expr::AtMost(_, v)
-                            | Expr::Greater(v)
-                            | Expr::Less(v)
-                            | Expr::StrLess(v)
-                            | Expr::StrLessEq(v)
-                            | Expr::Equal(v) => {
-                                stack.push(CExpr::End);
-                                for i in v.iter().rev() {
-                                    stack.push(CExpr::Expr(i));
-                                }
-                            }
-                            Expr::Strlen(e) | Expr::Not(e) => {
-                                stack.push(CExpr::End);
-                                stack.push(CExpr::Expr(e));
-                            }
-                            Expr::StrConcat(rhs, lhs) => {
-                                stack.push(CExpr::End);
-                                stack.push(CExpr::Expr(rhs));
-                                stack.push(CExpr::Expr(lhs));
-                            }
-                            Expr::Ite(cond, lhs, rhs) => {
-                                stack.push(CExpr::End);
-                                stack.push(CExpr::Expr(rhs));
-                                stack.push(CExpr::Expr(lhs));
-                                stack.push(CExpr::Expr(cond));
-                            }
-                            Expr::Bool(..) | Expr::String(..) | Expr::Real(..) | Expr::Var(..) => {}
+                        Expr::Strlen(e) | Expr::Not(e) => {
+                            stack.push(CExpr::End);
+                            stack.push(CExpr::Expr(e));
+                        }
+                        Expr::StrConcat(rhs, lhs) => {
+                            stack.push(CExpr::End);
+                            stack.push(CExpr::Expr(rhs));
+                            stack.push(CExpr::Expr(lhs));
                         }
+                        Expr::Ite(cond, lhs, rhs) => {
+                            stack.push(CExpr::End);
+                            stack.push(CExpr::Expr(rhs));
+                            stack.push(CExpr::Expr(lhs));
+                            stack.push(CExpr::Expr(cond));
+                        }
+                        Expr::Bool(..) | Expr::String(..) | Expr::Real(..) | Expr::Var(..) => {}
```

Comment:
what change messes with git here? most of it is similar but it seems to recognize it as lot of changes.

Reply:
We have outsourced a method into several methods, because in some cases we needed only parts of the method.We outsourced a method into several methods because in some cases we needed only parts of the method. Therefore, many changes are detected, although the code remains almost identical.